inc/wp-components/traits/trait-author.php

```diff
@@ -87,6 +87,30 @@ public function get_author_display_name() : string {
 		return $display_name;
 	}
 
+	/**
+	 * Get the author email address.
+	 *
+	 * @return string
+	 */
+	public function get_author_email() : string {
+		$email_address = '';
+
+		switch ( $this->get_author_type() ) {
+			case 'wp_user':
+				$email_address = $this->user->data->user_email;
```

Comment:
I'd add a fallback to `null` in case the trait is used in a component the author or correct post type is not set yet.